content/docs/command-reference/import-db.md

```diff
@@ -0,0 +1,153 @@
+# import-db
+
+Snapshot SQL query results from a database into CSV/JSON format.
+
+```usage
+usage: dvc import-db [-h] [-q | -v]
+            [--sql SQL] [--conn [CONNECTION]]
+            [--output-format [{csv,json}]] [-o [<path>]] [-f]
+```
+
+## Description
+
+With `import-db`, you can snapshot your ETL/database to a file to use in your
+data pipelines. This commands supports importing your sql query results into
+different file formats. To do so, you have to set connection strings to connect
+to a database to run your SQL query. The connection strings can be setup in
+config as `db.<name>`. Check [Database Connections](#database-connections) for
+more information.
+
+At the moment, `import-db` supports two different output format:
+
+- json records
+- csv (with header, and no index)
+
+An _import `.dvc` file_ is created in the same location e.g.
+`customers.txt.dvc`. This makes it possible to update the import later, if the
+data source has changed (see `dvc update`).
+
+<admon type="info">
+
+You can `dvc push` and `dvc pull` data imported from the databases to/from
+remote storage normally.
+
+</admon>
+
+## Database Connections
+
+For downloading sql query results with `--sql`, dvc needs to use a database
+connection string to connect to a database. This has to be configured in the
+[`db`] section.
+
+```dvc
+dvc config db.pgsql.url postgresql://user@hostname:port/database
+dvc config --local db.pgsql.password password
+```
+
+<admon type="warn" title="Security Alert">
+Configure password in a local config file, separately from the connection string.
+</admon>
+
+You can pass `--conn <name>` in addition with `--sql <query>` to use that
+database connection.
+
+```dvc
+dvc import-db --sql 'select * from table' --conn pgsql
+```
+
+In addition to a connection string, DVC needs a driver to connect to the
+database. Check [Installing database drivers](#installing-database-drivers) for
+connection string format and necessary driver for your specific database.
+
+[`db`]: /doc/user-guide/project-structure/configuration#db
+
+## Installing database drivers
+
+DVC does not come preinstalled with all the drivers for databases, you’ll need
+to install the required packages for the database you want to use.
+
+Some of the recommended packages are shown below, with their expected connection
+strings:
+
+| **Database**        | **PyPI package**                  | **Connection String**                                                                               |
```

Comment:
This is nice -- I didn't expect to document all the connection strings for different backends. Should we also include links to https://docs.sqlalchemy.org/en/20/core/engines.html#backend-specific-urls and/or https://docs.sqlalchemy.org/en/20/dialects/index.html?

Reply:
I have added a link to the former one.